meetings/overlay_mm_17_feb_2021.adoc

```diff
@@ -0,0 +1,63 @@
+== General summary
+* We will keep the meeting on the same time slot:
+*_Wednesday 6:00 am PST/2 pm UK/4 pm ISR/10 pm China_*
+* After HLD review will be complete we will start working on publish patches
+to llvm and GNU.
+
+== Action items
+* 2.7.1.5 add bullets
+* 2.7.2 elaborate that *ovliput* are sections arrived from *overlay attribute*?
+* 2.7.2.1 /2 lets mentioned that grouping tool is optional and number '3' is
+the default
+* 2.7.2.2
+** add dot to *ovlgrps*
+** fix bullets
+** add 'dot' to *ovlchache*
+** mention that *debug info* it should go into this section. On Debug chapter
+we will elaborate more.
+* 2.7.2.3
+** add 'dot' to *ovlplt* and move it to 2.7.2.2
+** add a cross-link to the syntax of the tables + explain again that it
+goes to group -1-
+* 2.7.2.4 add a cross-link to the *token table*
+* 2.7.2.5
+** add cross ref to group structure
+** flag to activate the creation of CRC checksum on linker side
+** let us state the chosen CRC algo (just pick one)
+** We need better names for MIN/MAX_GROUP_SIZE, and we need to align it with the
+draft.
+
+* Check if we had crc32 hook on the hook section. If missing, we should add it.
+** *[OS]*: The CRC is not part of the *HLD* same as all hooks. They are part of
+the implementation (*LLD*). On the _riscv-overlay-software-standard-draft.adoc_
+we only specified the *must* have hooks. CRC hook is an optional per-application
+design.

```

Comment:
Might be worth adding a short an explanation of why the crc32 is there, what it is the crc32 of and who uses it. Either in the requirements document or in HLD 2.2.1.

Reply:
it is only mentioned in the hld toolchain section, right ?
If so I think we should remove it

Reply:
The first reference to crc32 is in Figure 2 in the RT-Engine HLD.

Reply:
You are right. It is in the picture, that's why I did not find it.
I will update the picture.